db/migrate/20130827094554_alchemy_two_point_six.rb

```diff
@@ -310,6 +310,88 @@ def up
         t.string "name"
       end
     end
+  end
+
+  def down
+    if table_exists?('alchemy_attachments')
+      drop_table "alchemy_attachments"
+    end
+
+    if table_exists?('alchemy_cells')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.